src/config.js

```diff
@@ -0,0 +1,8 @@
+const env = window.AMP_CONFIG || {};
+
+export const urls = {
+  thirdParty: env.THIRD_PARTY_URL || 'https://3p.ampproject.net',
```

Comment:
I think we should keep the `AMP_CONFIG` variable "flat" for now rather than implementing a `defaultsDeep` mechanism like this:

```javascript
export const urls = defaultsDeep( window.AMP_CONFIG || {}, {
  urls: {
    ...
  }
});
```

That way, we can just do simple `val: val1 || val2`. I will go ahead and change the expected variables to `window.AMP_CONFIG.thirdPartyUrl` if that works for you. Or we can go the deeper structure route. Good either way.